### ISSUE TITLE
use pyimport_conda

### ISSUE THREAD

**Issue by stevengj** [jverzani/SymPy.jl#72]
As of PyCall 1.5, you can now just do `pyimport_conda("sympy", "sympy")` to auto-install SymPy via Conda (if PyCall is using Conda) or print an informative error message, rather than duplicating that logic here.   This will also let you remove the explicit Conda dependency.   Similarly for `mpmath`.

**Comment by jverzani** [comment]
Great. Thanks for taking the time to point this out!

**Comment by jverzani** [comment]
closed with 7103582a66213b51699ad78631cd1e29a23911d6

**Comment by stevengj** [comment]
@jverzani, actually it is is a lot simpler than your b47d11288aa9ec39ae243cf837f367fe0c9cf5fb

The code in SymPy.jl should just be:
```jl
function __init__()
    ## Define sympy, mpmath, ...
    copy!(sympy, pyimport_conda("sympy", "sympy"))

    ...
end
```
The whole `try .. catch` is implemented by `pyimport_conda`.
